resources/settings/innovo-inventor.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "innovo-inventor",
+    "name": "Innovo INVENTOR",
+    "manufacturer": "INNOVO",
+    "author": "AR",
+    "version": 1,
+    "icon": "",
+    "platform": "inventor_platform.stl",
+    "platform_texture": "",
+    "inherits": "fdmprinter.json",
+    "machine_settings": {
+        "machine_width": {"default": 340},
+        "machine_height": {"default": 290},
+        "machine_depth": {"default": 300},
+        "machine_center_is_zero": {"default": false},
+        "machine_nozzle_size": {"default": 0.4},
+        "machine_head_shape_min_x": {"default": 43.7},
+        "machine_head_shape_min_y": {"default": 19.2},
+        "machine_head_shape_max_x": {"default": 43.7},
+        "machine_head_shape_max_y": {"default": 55},
+        "machine_nozzle_gantry_distance": {"default": 82.3},
+        "machine_nozzle_offset_x_1": {"default": 0},
+        "machine_nozzle_offset_y_1": {"default": 15},
+        "machine_gcode_flavor": {"default": "RepRap (Marlin/Sprinter)"},
+        "machine_start_gcode": {"default": "G28 ; Home extruder\nM107 ; Turn off fan\nG90 ; Absolute positioning\nM82 ; Extruder in absolute mode\n{IF_BED}M190 S{BED}\n{IF_EXT0}M104 T0 S{TEMP0}\n{IF_EXT0}M109 T0 S{TEMP0}\n{IF_EXT1}M104 T1 S{TEMP1}\n{IF_EXT1}M109 T1 S{TEMP1}\nG32 S3 ; auto level\nG92 E0 ; Reset extruder position"},
+        "machine_end_gcode": {"default": "M104 S0\nG91 ; relative positioning\nG1 E-2 F5000; retract 2mm\nG28 Z; move bed down\nG90 ; absolute positioning\nM84   ; disable motors"}
+    },
+    
+    "overrides": { 
+        "layer_height": { "default": 0.15 },
+        "shell_thickness": { "default": 0.8},
+        "wall_thickness": { "default": 0.8 },
+        "top_bottom_thickness": { "default": 0.3, "visible": true },
+        "material_print_temperature": { "default": 215, "visible": true },
+        "material_bed_temperature": { "default": 60, "visible": true },
```

Comment:
You're right.  I wasn't intending to have the heated bed installed but was
asked to make it visible even if it was not used.  However, I now plan to
have the heated bed installed. So I will change this right away.

*Adam Rumjahn*

* <http://www.innovo3d.co>*

*adam@innovo3d.co <adam@innovo3d.co>Cel: +57 304 643 1248*
*Fijo: +57 2 374 2473*

On Sun, Sep 13, 2015 at 2:26 PM, awhiemstra <notifications@github.com>
wrote:

> In resources/settings/innovo-inventor.json
> <https://github.com/Ultimaker/Cura/pull/301#discussion_r39350957>:
>
> > +        "machine_head_shape_max_y": {"default": 55},
> > +        "machine_nozzle_gantry_distance": {"default": 82.3},
> > +        "machine_nozzle_offset_x_1": {"default": 0},
> > +        "machine_nozzle_offset_y_1": {"default": 15},
> > +        "machine_gcode_flavor": {"default": "RepRap (Marlin/Sprinter)"},
> > +        "machine_start_gcode": {"default": "G28 ; Home extruder\nM107 ; Turn off fan\nG90 ; Absolute positioning\nM82 ; Extruder in absolute mode\n{IF_BED}M190 S{BED}\n{IF_EXT0}M104 T0 S{TEMP0}\n{IF_EXT0}M109 T0 S{TEMP0}\n{IF_EXT1}M104 T1 S{TEMP1}\n{IF_EXT1}M109 T1 S{TEMP1}\nG32 S3 ; auto level\nG92 E0 ; Reset extruder position"},
> > +        "machine_end_gcode": {"default": "M104 S0\nG91 ; relative positioning\nG1 E-2 F5000; retract 2mm\nG28 Z; move bed down\nG90 ; absolute positioning\nM84   ; disable motors"}
> > +    },
> > +
> > +    "overrides": {
> > +        "layer_height": { "default": 0.15 },
> > +        "shell_thickness": { "default": 0.8},
> > +        "wall_thickness": { "default": 0.8 },
> > +        "top_bottom_thickness": { "default": 0.3, "visible": true },
> > +        "material_print_temperature": { "default": 215, "visible": true },
> > +        "material_bed_temperature": { "default": 60, "visible": true },
>
> You're setting Bed Temperature to visible but not setting the machine
> setting "machine_heated_bed" to true. This will mean you will not get the
> right heated bed behaviour since several parts of the code check the
> machine_heated_bed setting.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Ultimaker/Cura/pull/301/files#r39350957>.
>